maintainers/maintainer-list.nix

```diff
@@ -16010,6 +16010,12 @@
     githubId = 920910;
     name = "peelz";
   };
+  peigongdsd = {
+    email = "peigong2013@outlook.com";
+    github = "peigongdsd";
+    githubId = 	51317171;
```

Comment:
There is an extra space in the githubId line

Reply:
Thanks